fastlane/Fastfile

```diff
@@ -78,6 +79,71 @@ lane :release do |options|
   github_release(version: version_number)
 end
 
+desc "Creates GitHub snapshot pre-release and publishes with the 'snapshot' tag"
+lane :snapshot_release do
+  stable_version   = current_version_number
+  snapshot_version = "#{stable_version}-SNAPSHOT.#{Time.now.strftime('%Y%m%d%H%M')}"
+
+  UI.message("Creating snapshot release #{snapshot_version}...")
+
+  Dir.chdir(get_root_folder) do
+    sh("npm version #{snapshot_version} --no-git-tag-version")
+    sh("npm publish --tag snapshot")
+    sh("npm version #{stable_version} --no-git-tag-version")
+  end
+
+  add_git_tag(tag: snapshot_version, message: "Snapshot #{snapshot_version}")
+  push_to_git_remote(remote: "origin", tags: true)
+
+  create_github_release(
```

Comment:
Hmm I don't know if we want to create github releases for snapshots... It could get really noisy. Wdyt?